src/RecoveryServices/RecoveryServices.SiteRecovery/NetworkInterfaceCard/NewAzureRmAsrVmNicConfig.cs

```diff
@@ -162,7 +162,7 @@ public class NewAzureRmAsrVmNicConfig : SiteRecoveryCmdletBase
             Mandatory = false,
             HelpMessage = "Specifies test failover/failover settings of NIC IP configs.")]
         [ValidateNotNull]
-        public IPConfigInputDetails[] IPConfig { get; set; }
+        public PSIPConfigInputDetails[] IPConfig { get; set; }
```

Comment:
Could you share the PS design review PR link? Wanted to go through that once.

Reply:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/886